spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/DefaultBinding.java

```diff
@@ -95,6 +96,20 @@ public final synchronized void stop() {
 			this.lifecycle.stop();
 		}
 	}
+	
+	@Override
+	public final synchronized void pause() {
+		if (this.lifecycle instanceof Pausable) {
+			((Pausable)this.lifecycle).pause();
```

Comment:
I wonder if the actuator should return an error if it's not a pausable binding?

Reply:
may be a WARN for now. If we return something now we'll set the expectations and I am not ready for it yet